### ISSUE TITLE
De-activate automatic updats and change homeserver in FatClients 

### ISSUE THREAD

**Issue by AnassDriate** [vector-im/element-web#24860]
### Your use case

Is there a configuration to de-activate the automatic updats in fat-clients ?

Is there a configuration to change the homeserver in fatclients?
(let me clarify this point : as an administrator i want the group A to use only the server testa.com and the group B to use only the server testb.com and i do not want to build two different version of the app )

**Comment by t3chguy** [comment]
> Is there a configuration to de-activate the automatic updats in fat-clients ?

Yup - https://github.com/vector-im/element-desktop/#config `"update_base_url": null` or pass `--no-update` to the executable each time you launch it

Same goes for homeserver configuration - https://github.com/vector-im/element-web/blob/develop/docs/config.md
